net/ftpproxy/src/opnsense/mvc/app/controllers/OPNsense/FtpProxy/forms/dialogEdit.xml

```diff
@@ -0,0 +1,92 @@
+<form>
+   <field>
+      <id>ftpproxy.enabled</id>
+      <label>enabled</label>
+      <type>checkbox</type>
+      <help><![CDATA[Enable or disable this ftp proxy.]]></help>
+   </field>
+   <field>
+      <id>ftpproxy.listenaddress</id>
+      <label>Listen address</label>
+      <type>text</type>
+      <help><![CDATA[Address where the proxy will listen for redirected control
+                     connections. The default is 127.0.0.1.]]></help>
```

Comment:
You could remove the line feeds with a regexp before you translate it.
preg_replace('/\r?\n|\r/g', ' ', $string);
Sorry, but I've no idea how the translation is done.